pkgs/by-name/wi/winchecksec/package.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pe-parse,
+  uthenticode,
+  openssl,
+  gtest,
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
Prefer `finalAttrs` over `rec`
- https://github.com/NixOS/nixpkgs/issues/315337

---

Thank you for the swift review! I also appreciate your including these tracking issues, as I was aware of both approaches but I didn't know when to prefer `finalAttrs` to `rec`.